UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -554,20 +529,13 @@ private void SetConsciousState(ConsciousState NewConsciousState)
 		{
 			if (damageSplit)
 			{
-				float bodyParts = RootBodyPartContainers.Sum(Container => Container.ContainsLimbs.Count());
+				float bodyParts = BodyPartList.Count;
 				damage /= bodyParts;
 			}
 
-			foreach (var Container in RootBodyPartContainers)
+			foreach (var bodyPart in BodyPartList)
```

Comment:
You can't use BodyPartList since that would include organs as well, you need to do damage to the root body parts to propagate the damage down properly through the different armour of the body parts

Reply:
BodyPartList doesnt include organs anymore

Reply:
how?

Reply:
I'm just not adding organs to this list anymore